go/flags/endtoend/zkctld.txt

```diff
@@ -1,26 +1,7 @@
-      --config-persistence-min-interval duration                    minimum interval between persisting dynamic config changes back to disk (if no change has occurred, nothing is done). (default 1s)
-      --config-type string                                          Config file type (omit to infer config type from file extension).
-  -h, --help                                                        display usage and exit
-      --keep_logs duration                                          keep logs for this long (using ctime) (zero to keep forever)
-      --keep_logs_by_mtime duration                                 keep logs for this long (using mtime) (zero to keep forever)
-      --log_backtrace_at traceLocation                              when logging hits line file:N, emit a stack trace (default :0)
-      --log_dir string                                              If non-empty, write log files in this directory
-      --log_err_stacks                                              log stack traces for errors
-      --log_rotate_max_size uint                                    size in bytes at which logs are rotated (glog.MaxSize) (default 1887436800)
-      --logtostderr                                                 log to standard error instead of files
-      --pprof strings                                               enable profiling
-      --purge_logs_interval duration                                how often try to remove old logs (default 1h0m0s)
-      --security_policy string                                      the name of a registered security policy to use for controlling access to URLs - empty means allow all for anyone (built-in policies: deny-all, read-only)
-      --stderrthreshold severity                                    logs at or above this threshold go to stderr (default 1)
-      --v Level                                                     log level for V logs
-  -v, --version                                                     print binary version
-      --vmodule moduleSpec                                          comma-separated list of pattern=N settings for file-filtered logging
-      --zk.cfg string                                               zkid@server1:leaderPort1:electionPort1:clientPort1,...) (default "6@<hostname>:3801:3802:3803")
-      --zk.extra stringArray                                        extra config line(s) to append verbatim to config (flag can be specified more than once)
-      --zk.myid uint                                                which server do you want to be? only needed when running multiple instance on one box, otherwise myid is implied by hostname
+zkctld is a daemon that starts or initializes ZooKeeper with Vitess-specific configuration. It will stay running as long as the underlying ZooKeeper server, and will pass along SIGTERM.
+
+Usage:
+  zkctld [flags]
+
```

Comment:
Why isn't there an `Available Commands:` section for this binary?

Reply:
because there are no subcommands https://github.com/vitessio/vitess/blob/main/go/cmd/zkctld/zkctld.go#L61